Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -73,13 +71,30 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
         tableView.cellLayoutMarginsFollowReadableWidth = false
         tableView.accessibilityIdentifier = "Context Menu"
 
+        let width = min(self.view.frame.size.width, ActionOverlayTableViewUX.MaxWidth) - (ActionOverlayTableViewUX.Padding * 2)
+
         tableView.snp.makeConstraints { make in
             make.center.equalTo(self.view)
-            make.width.equalTo(290)
+            make.width.equalTo(width)
             setHeightConstraint(make)
         }
     }
 
+    private func applyBackgroundBlur() {
+        let appDelegate = UIApplication.shared.delegate as! AppDelegate
+        if let screenshot = appDelegate.window?.screenshot() {
+            let blurredImage = screenshot.applyBlur(withRadius: 5,
```

Comment:
By using a screenshot to do the blurring, when the user rotates the device the blur background gets all mixed up because the screenshot was taken from the previous orientation. Could we use the UIVisualEffectView here to blur the background dynamically so we don't need to take the screenshot for the blur?